[2021-03-23 16:00:49,243] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: data_prep_and_process.data_processing 2021-03-23T10:17:17.705601+00:00 [queued]>
[2021-03-23 16:00:49,252] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: data_prep_and_process.data_processing 2021-03-23T10:17:17.705601+00:00 [queued]>
[2021-03-23 16:00:49,252] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-23 16:00:49,253] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2021-03-23 16:00:49,253] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-23 16:00:49,265] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): data_processing> on 2021-03-23T10:17:17.705601+00:00
[2021-03-23 16:00:49,269] {standard_task_runner.py:52} INFO - Started process 45660 to run task
[2021-03-23 16:00:49,281] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_prep_and_process', 'data_processing', '2021-03-23T10:17:17.705601+00:00', '--job-id', '57', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/master_dag.py', '--cfg-path', '/var/folders/yw/952dv3rd0332_2c7vqyq34vc0000gp/T/tmp55cq9qq1', '--error-file', '/var/folders/yw/952dv3rd0332_2c7vqyq34vc0000gp/T/tmp5d_mhcqu']
[2021-03-23 16:00:49,285] {standard_task_runner.py:77} INFO - Job 57: Subtask data_processing
[2021-03-23 16:00:54,350] {logging_mixin.py:104} INFO - Running <TaskInstance: data_prep_and_process.data_processing 2021-03-23T10:17:17.705601+00:00 [running]> on host FACT-LAP-135.local
[2021-03-23 16:00:59,443] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=skadhithya95@example.com
AIRFLOW_CTX_DAG_OWNER=Adhithya_Karthikeyan
AIRFLOW_CTX_DAG_ID=data_prep_and_process
AIRFLOW_CTX_TASK_ID=data_processing
AIRFLOW_CTX_EXECUTION_DATE=2021-03-23T10:17:17.705601+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-23T10:17:17.705601+00:00
[2021-03-23 16:00:59,451] {data_processing.py:40} INFO - SPARK df --> Reading JSON file from path
[2021-03-23 16:00:59,453] {data_processing.py:41} INFO - /Users/adhithyakarthikeyan/currency_hist_records.json
[2021-03-23 16:01:09,191] {data_processing.py:43} INFO - Status --> Successfully

[2021-03-23 16:01:09,193] {data_processing.py:50} INFO - Removing duplicate values from df
[2021-03-23 16:01:09,390] {data_processing.py:52} INFO - Printing the df 
[2021-03-23 16:01:14,486] {logging_mixin.py:104} INFO - +----------+--------+-------+
|      date|currency|   rate|
+----------+--------+-------+
|2018-11-30|     RON| 4.6598|
|2020-05-15|     HKD| 8.3693|
|2018-02-13|     TRY| 4.6865|
|2018-08-16|     GBP|0.89498|
|2020-10-09|     NZD| 1.7821|
+----------+--------+-------+
only showing top 5 rows
[2021-03-23 16:01:14,500] {logging_mixin.py:104} INFO - 
[2021-03-23 16:01:14,544] {data_processing.py:78} INFO - Pushing the tables to Postgres SQL using JDBC Pyspark
[2021-03-23 16:01:34,391] {data_processing.py:89} INFO - Pushed the tables successfully
[2021-03-23 16:01:34,395] {python.py:118} INFO - Done. Returned value was: <data_processing.data_processing object at 0x7fe71ac01d60>
[2021-03-23 16:01:34,435] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=data_prep_and_process, task_id=data_processing, execution_date=20210323T101717, start_date=20210323T103049, end_date=20210323T103134
[2021-03-23 16:01:34,475] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-23 16:01:34,506] {local_task_job.py:146} INFO - Task exited with return code 0
